R/ard_continuous.R

```diff
@@ -137,6 +137,8 @@ ard_continuous <- function(data,
     dplyr::select(all_ard_groups(), "...ard_all_stats...") |>
     tidyr::unnest(cols = "...ard_all_stats...")
 
+  # is this a duplication?
```

Comment:
@ddsjoberg is the code from lines 135-138 (above this comment I added) and 143-146 (below) an accidental duplicate or were they suppose to contain different portions of codes that perform different things (based on the comments)

Reply:
I think the first comment "# add default function labels and formatters" was leftover from some reworking of the organization. I deleted the comment, thanks!

![image](https://github.com/insightsengineering/cards/assets/26774684/721a7772-6785-4001-9a95-46beba7c1c30)